pkg/app/piped/platformprovider/ecs/client.go

```diff
@@ -29,9 +30,17 @@ import (
 	"go.uber.org/zap"
 
 	"github.com/pipe-cd/pipecd/pkg/app/piped/platformprovider"
+	"github.com/pipe-cd/pipecd/pkg/backoff"
 	appconfig "github.com/pipe-cd/pipecd/pkg/config"
 )
 
+const (
+	// Retry wait service stable check.
+	retryServiceStable = 40
+	// Interval wait service stable check.
+	retryServiceStableInterval = 15 * time.Second
```

Comment:
Can you tell me why this timeout is 10mins(40 * 15sec)?
The default deployment timeout is now 6 hours.

Reply:
This follows the AWS cli implementation, where they use the above numbers. Of course, we can wait up to 6 hours, but I just feel like it's overdoing.
https://awscli.amazonaws.com/v2/documentation/api/latest/reference/ecs/wait/services-stable.html#description

Reply:
OK 👍 Thanks